LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -1,7 +1,7 @@
 <div class="leaf-center-content">
 
     <!-- LEFT SIDE NAV -->
-    <!--{assign var=left_nav_content value="
+    <!--{assign var=left_uag_content value="
```

Comment:
Yeah so the left_nav_content gets used in other areas such as template editor and so forth so I made the UAG navs and added into the css so its unique to the UAG only as this will be used later for redesign as well. Later on during the redesign everything will probably change anyway and we will remove most of the left-nav-content class's as we reconstruct most of the admin panel stuff.

Reply:
Sounds good, that was my only question.